README.md

```diff
@@ -41,7 +41,9 @@ Before we deploy a devnet we will publish a deployment manifest, including:
   - Contracts versions (can be a sha or tag)
   - Configuration flags that need to be set on node software
 
-When we make changes to devnets, including bug fix deployments, etc we will make srue to update this repo so external users can follow the upgrades.
+When we make changes to devnets, including bug fix deployments, etc we will update this file so external users can follow the upgrades.
```

Comment:
This is likely implicit, but we should have consistency checks between the spec and the deployment, so that we don't fall out of sync